### PR TITLE
Moving the status bar to the toolbar

### PR DESCRIPTION
**Pull request by shtukas**

### CODE REVIEW

**Comment by shtukas** [comment]
Before: 

picture 1

**Comment by shtukas** [comment]
After: 

picture 2

**Comment by shtukas** [comment]
Deployed to CODE.

File: explainer-server/app/views/explainEditor.scala.html [hunk 1]
```diff
@@ -48,14 +51,23 @@
...
+            if (getBodyWordCount() > maxWordCount) {
                 $(".top-toolbar__word-count-wrapper__warning-outer").show();
-            }else{
+            } else {
                 $(".top-toolbar__word-count-wrapper__warning-outer").hide();
             }
         }
 
+        function updateStatusBar(message){
+            if(message.length>0){
```
Comment (shaundillon) [inline explainer-server/app/views/explainEditor.scala.html:63]: This could be simplified to `if(message.length) {`... JS evaluates `0` to `false`.

Comment (shtukas) [inline explainer-server/app/views/explainEditor.scala.html:63]: Yes. I guess it's having done Ruby, where anything non `null` or `false` evaluates to `true` :)

I like being explicit, so I would vote to leave it like that (unless you feel strongly about it ^^)

**Comment by philmcmahon** [comment]
👍 for code

**Pull request merged**